backend/experiment/models.py

```diff
@@ -213,6 +212,7 @@ def export_table(self, session_keys: List[str], result_keys: List[str], export_o
 
     def get_rules(self):
         """Get instance of rules class to be used for this session"""
+        from experiment.rules import EXPERIMENT_RULES
```

Comment:
why this move? Was there a circular import problem?

Reply:
Yes :-( I'm importing and using the `Experiment` class in the CongoSameDiff ruleset, which caused the circular import issues. Do you know any other way to solve it?

Reply:
No, I was just curious why we didn't have this problem before. The answer is: we didn't add type hints before.